chapters/ch10.md

```diff
@@ -4,25 +4,25 @@
 
 > It is very difficult to know in advance what approach will work best for a new problem. Even experienced machine learning researchers will usually try out many dozens of ideas before they discover something satisfactory. When building a machine learning system, I will often:
 
-Thật sự rất khó để đoán trước phương án tiếp cận nào tốt nhất cho một vấn đề mới. Kể cả những nhà nghiên cứu học máy dày dặn kinh nghiệm cũng thường thử nghiệm cả chục ý tưởng mới khám phá ra cái gì đó thỏa mãn. Khi xây dựng một hệ thống học máy, tôi thường: 
+Thật sự rất khó để biết trước phương án tiếp cận nào là tốt nhất cho một vấn đề mới. Kể cả những nhà nghiên cứu học máy dày dặn kinh nghiệm cũng thường thử nghiệm cả chục ý tưởng mới khám phá ra một hướng đi thoả đáng. Khi xây dựng một hệ thống học máy, tôi thường: 
 
 > 1. Start off with some **idea** on how to build the system.
 
-1. Bắt đầu với một vài ý tưởng xây dựng hệ thống.
+1. Bắt đầu bằng một vài *ý tưởng* về cách xây dựng hệ thống đó.
```

Comment:
bỏ từ "đó"

Reply:
"đó" ở đây mình dịch cho chữ "the system", khác với "a system" trong câu trên.

Reply:
Từ “đó” không thể hiện là system nào và cũng yêu cầu ngầm hiểu của người đọc. Khác với từ “trên” — đã trình bày phía trên.